covid19_rsid_export.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python
+import argparse
+import datetime
+import json
+import os
+import pymongo
+
+from ebi_eva_common_pyutils.mongo_utils import get_mongo_uri_for_eva_profile
+from pymongo import MongoClient
+from pymongo.read_concern import ReadConcern
+from pymongo.read_preferences import ReadPreference
+from typing import List
+
+
+timestamp = datetime.datetime.now().strftime("%Y%m%d%H%M%S")
+
+BATCH_SIZE = 1000
+ACCESSIONING_DATABASE_NAME = "eva_accession_sharded"
+RELEASE_RECORD_COLLECTION_NAME = "releaseRecordEntity"
+COVID19_ASSEMBLY_ACCESSION = "GCA_009858895.3"
```

Comment:
I would set that variable as a command line argument with a default value in case we want to change it to another assembly.